src/pages/en/guides/debugging.md

```diff
@@ -1,16 +1,94 @@
 ---
 layout: ~/layouts/MainLayout.astro
 title: Debugging
-description: Debug in Astro using the Debug component
+description: Need help? Stuck on something? Got you covered.
 ---
 
 Astro provides several different tools to help you debug your code easier and faster.
 
-## Debugging with `console.log()`
+## Common Error Messages
 
-`console.log()` is a simple-but-popular method of debugging your Astro code. Where you write your `console.log` statment will determine where your debugging output is printed. 
+### Transform failed with X error
 
-### Frontmatter
+In the majority of cases, this is due to your imports not being at the top of your `.astro` file, check if they are.
+
+If it's not the case, [ask us on discord](#need-more)
+
+💡 Not sure? Check if [you're not the only one](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Transform+failed+with+*+error) with this issue!
+
+### Cannot use import statement outside a module
+
+In Astro components, `<script>` tags are by default loaded as [JS modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules). Be careful, it's this behavior will be disabled if `type="module"` or any other attribute is added to the tag. Same as applying the [`is:inline` directive](/en/reference/directives-reference#isinline).
+
+You choose to set the `is:inline` directive don't forget to also add the `type="module"` attribute to the script to be able to use import statements.
+
+💡 Not sure? Check if [you're not the only one](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Cannot+use+import+statement) with this issue!
+
+### Unable to render Component
+
+This is usually due to errors in your component, check the corresponding documentation for your component and make sure it's working correctly.
+
+> This can also be due from accessing the `window` or `document` object at render time. By default, Astro will render your component outside of the browser (exept for the `client:only` directive). Try to access those objects after rendering (ex: [`useEffect()`](https://reactjs.org/docs/hooks-reference.html#useeffect) in react or [`onMounted()`](https://vuejs.org/api/composition-api-lifecycle.html#onmounted) in vue and svelte)
+
+💡 Not sure? Check if [you're not the only one](https://github.com/withastro/astro/issues?q=is%3Aissue+is%3Aopen+Unable+to+render+Component) with this issue!
+
+### Export statements must be placed at the top of `.astro` files!
+
+When dealing with Astro components, always check if your `import` and `export` statements are at the top of your component script.
+
+Unless you're using relatives paths using the `import()` function.
```

Comment:
I was mentioning more the common "issue" when people forgot to put their export / import not at the top of the file

I think we should avoid mentioning current issues because it'll be difficult to maintain like said in #440